griptape/drivers/vector/astra_db_vector_store_driver.py

```diff
@@ -0,0 +1,184 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Optional
+
+from attrs import define, field
+
+from griptape.drivers import BaseVectorStoreDriver
+from griptape.utils import import_optional_dependency
+
+if TYPE_CHECKING:
+    from astrapy import Collection
+    from astrapy.authentication import TokenProvider
+
+
+@define
+class AstraDBVectorStoreDriver(BaseVectorStoreDriver):
```

Comment:
Ah let's rename to `AstraDbVectorStoreDriver` (`astradb_vector_store_driver.py`) for consistency with other Drivers like `AmazonDynamoDbConversationMemoryDriver` and `MongoDbAtlasVectorStoreDriver`.

Reply:
Agree; I think our all-uppercase "DB" as per branding guidelines is indeed a weaker principle than internal naming consistency within Griptape.

Reply:
@hemidactylus sorry, last request: please rename `astra_db_vector_store_driver.py` to `astradb_vector_store_driver.py`.

Reply:
No problem :)